style.css

```diff
@@ -106,342 +139,420 @@ body {
+  width: 327px;
+  margin-top: 1rem;
+  display: flex;
+  outline: none;
+  gap: 1.5rem;
+  justify-content: center;
+  border-radius: 8px;
 }
 
-.about_btn {
-  width: 167px;
+input::placeholder {
   padding: 12px;
-  color: #7f8cff;
-  background-color: transparent;
-  border: 1px solid #7f8cff;
-  border-radius: 15px;
-  height: 48px;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  text-align: center;
-  letter-spacing: 0.03em;
-  cursor: pointer;
+  color: #172b4d;
 }
 
-.about_btn:hover {
-  border: 1px #7f8cff solid;
-  color: #000;
-  background-color: #f4f5f728;
+textarea {
+  width: 350px;
+  height: 180px;
+  margin-top: 1.5rem;
+  border-radius: 8px;
+  border: none;
 }
 
-/* about languages section */
-.about-languages {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 10rem;
-  justify-content: space-between;
-  margin-top: 100px;
+textarea::placeholder {
+  padding: 15px;
+  color: #b3bac5;
 }
 
-.lang-title {
-  display: flex;
-  row-gap: 17px;
-  align-items: center;
-  justify-content: flex-start;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 1.7;
+/* button */
+button {
+  display: block;
+  color: #396df2;
+  background-color: white;
+  border: 1px solid #6070ff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 8px;
+  align-self: flex-start;
+  padding: 10px;
+  transition: all ease-in-out 1s;
 }
 
-.arrow-down {
-  display: flex;
-  justify-content: flex-end;
-  align-self: center;
-  margin-left: 18rem;
+form button {
+  margin-bottom: 20px;
 }
 
-.image-language {
-  position: relative;
-  width: auto;
-  height: 200px;
-  background-color: #fff;
-  margin-bottom: 5rem;
-  padding: 0 5px  3px 0;
+button:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: white;
 }
 
-.skills-language > h3 {
-  font-size: 20px;
-  font-weight: 700;
-  color: #344563;
-  margin-bottom: 1rem;
+button:active {
+  color: white;
+  background: #2230d2;
+  box-shadow: none;
 }
 
-.arrow-right {
-  display: flex;
-  justify-content: end;
-  align-items: center;
-  margin-left: 18rem;
-  font-weight: normal;
+.about button {
+  margin: 12px 0 20px 0;
 }
 
-.skills-info {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  gap: 2.5rem;
+/* button */
+
+.text-input {
+  width: 327px;
+  height: 48px;
 }
 
-.skills-title {
-  font-size: 24px;
-  font-weight: 700;
-  color: #505f79;
-  display: flex;
-  justify-content: left;
-  margin-bottom: 3rem;
-  padding-top: 7px;
+.textarea {
+  width: 327px;
 }
 
-.arrow-side {
-  display: flex;
-  justify-content: flex-end;
-  align-self: center;
-  font-weight: normal;
+@media only screen and (max-width: 767px) {
+  .nav-link {
+    display: none;
+  }
+
+  button {
+    margin-top: 1rem;
+    display: block;
+  }
+
+  .card {
+    width: 80%;
+    height: 540px;
+    padding-left: 2rem;
+  }
+}
+
```

Comment:
- [ ] Your project cards no longer look like they should be on mobile devices. You could remove the `padding-left: 2rem` because it is currently pushing the card to the right causing your design to look skewed. Also find a way to fix the button on its own row. **Hint:** Flexbox. 😎 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/168428283-31cd3593-7561-46b8-b227-aa453c3f326b.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/168428320-3b5959d7-6f58-41cc-930a-88424d234215.png)